front/graph.send.php

```diff
@@ -46,7 +46,7 @@
    if (($userID == $uid)
        && file_exists(GLPI_GRAPH_DIR."/".$_GET["file"])) {
 
-      list($fname,$extension)=explode(".", $filename);
+      list( ,$extension)=explode(".", $filename);
```

Comment:
I really do not like this way to write the code. I'd prefer to keep `$fname` not used variable (ths does not really cost something).

Reply:
Done.